pydantic/utils.py

```diff
@@ -241,7 +241,8 @@ def unique_list(input_list: Union[List[T], Tuple[T, ...]]) -> List[T]:
 
 
 def update_normalized_all(
-    item: Union['AbstractSetIntStr', 'MappingIntStrAny'], all_items: Union['AbstractSetIntStr', 'MappingIntStrAny'],
+    item: Union['AbstractSetIntStr', 'MappingIntStrAny'],
```

Comment:
this doesn't need changing.

Reply:
PEP8 says that a line has to be less than 79 characters